test/parallel/test-runner-coverage.js

```diff
@@ -487,24 +502,25 @@ test('coverage with included and excluded files', skipIfNoInspector, () => {
 });
 
 test('properly accounts for line endings in source maps', skipIfNoInspector, () => {
-  const fixture = fixtures.path('test-runner', 'source-map-line-lengths', 'index.js');
+  const fixture = fixtures.path('test-runner', 'source-map', 'line-lengths', 'index.js');
   const args = [
     '--test', '--experimental-test-coverage', '--test-reporter', 'tap',
     fixture,
   ];
   const report = [
     '# start of coverage report',
-    '# ----------------------------------------------------------------------------',
-    '# file                        | line % | branch % | funcs % | uncovered lines',
-    '# ----------------------------------------------------------------------------',
-    '# test                        |        |          |         | ',
-    '#  fixtures                   |        |          |         | ',
-    '#   test-runner               |        |          |         | ',
-    '#    source-map-line-lengths  |        |          |         | ',
-    '#     index.ts                | 100.00 |   100.00 |  100.00 | ',
-    '# ----------------------------------------------------------------------------',
-    '# all files                   | 100.00 |   100.00 |  100.00 | ',
-    '# ----------------------------------------------------------------------------',
+    '# ------------------------------------------------------------------',
+    '# file              | line % | branch % | funcs % | uncovered lines',
+    '# ------------------------------------------------------------------',
+    '# test              |        |          |         | ',
+    '#  fixtures         |        |          |         | ',
+    '#   test-runner     |        |          |         | ',
+    '#    source-map     |        |          |         | ',
+    '#     line-lengths  |        |          |         | ',
+    '#      index.ts     | 100.00 |   100.00 |  100.00 | ',
+    '# ------------------------------------------------------------------',
+    '# all files         | 100.00 |   100.00 |  100.00 | ',
+    '# ------------------------------------------------------------------',
```

Comment:
**Review note**

I moved `source-map-line-lengths` into `source-map/line-lengths`, hence this change.